src/community/pom.xml

```diff
@@ -321,6 +320,21 @@
       <modules>
         <module>script</module>
       </modules>
+      <build>
+       <plugins>
+         <plugin>
+          <artifactId>maven-assembly-plugin</artifactId>
+          <version>3.2.0</version>
+          <configuration>
+            <descriptors>
+              <descriptor>release/ext-groovy.xml</descriptor>
+              <descriptor>release/ext-javascript.xml</descriptor>
+              <descriptor>release/ext-python.xml</descriptor>
```

Comment:
I don't think this is going to work how you want it to without significant revisions to src/community/release/pom.xml, since it is used for assembly and depends upon everything in communityRelease at the top level

Reply:
I saw the profile names used in release/pom.xml and used the same names here.